pcdsdevices/epics_motor.py

```diff
@@ -103,10 +103,13 @@ def format_status_info(self, status_info):
         status: str
             Formatted string with all relevant status information.
         """
-        description = get_status_value(status_info, 'description', 'value')
-        units = get_status_value(status_info, 'user_setpoint', 'units')
-        dial = get_status_value(status_info, 'dial_position', 'value')
-        user = get_status_value(status_info, 'position')
+        description = get_status_float(status_info, 'description', 'value',
+                                       precision=3)
```

Comment:
Is the description a float?

Reply:
Lol I guess I got carried away. The way `get_status_float` is written though, it functions exactly the same as `get_status_value` for non-floats.

Reply:
Would it be a bad idea to condense them to one function?

Reply:
I'm not motivated to refactor get_status things because I want to refactor the entire printout setup to use jinja, so if this doesn't cause a problem we can leave it